include/deal.II/grid/reference_cell.h

```diff
@@ -1881,36 +1874,35 @@ ReferenceCell::face_to_cell_vertices(const unsigned int  face,
         }
       case ReferenceCells::Line:
         {
+          const auto [face_orientation, face_rotation, face_flip] =
+            internal::split_face_orientation(combined_face_orientation);
+
           return GeometryInfo<1>::face_to_cell_vertices(
-            face,
-            vertex,
-            Utilities::get_bit(face_orientation, 0),
-            Utilities::get_bit(face_orientation, 2),
-            Utilities::get_bit(face_orientation, 1));
+            face, vertex, face_orientation, face_flip, face_rotation);
         }
       case ReferenceCells::Triangle:
         {
           static constexpr ndarray<unsigned int, 3, 2> table = {
             {{{0, 1}}, {{1, 2}}, {{2, 0}}}};
 
-          return table[face][face_orientation != 0u ? vertex : (1 - vertex)];
+          return table[face]
+                      [combined_face_orientation != 0u ? vertex : (1 - vertex)];
```

Comment:
Hm, is this how you want to do it in the future? I thought of `combined_face_orientation` as some sort of opaque representation of information that you can only access through specific functions. But here you use a specific value of that opaque representation -- that doesn't quite sound right. Perhaps there ought to be a function such as `is_standard_orientation(combined_face_orientation)` or some something like this?